test/parallel/test-stream-writableState-uncorked-bufferedRequestCount.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const stream = require('stream');
+
+const writable = new stream.Writable();
+
+writable._write = (chunk, encoding, cb) => {
```

Comment:
We should consider using a function here, as we make no guarentees that `apply()` or `call()` may not be called n this internally in the future.

(Assigning arrow functions as prototype functions is generally bad practice because those do not work.)